.coderabbit.yaml

```diff
@@ -28,6 +28,7 @@ reviews:
     issue_assessment:
       mode: warning
   auto_review:
+    base_branches: ["develop", "main", "release/.*"]
```

Comment:
I don't think main is strictly necessary since we will never(?) have a PR opened there that _isn't_ a release